roles/etcd/tasks/main.yml

```diff
@@ -45,10 +51,10 @@
     - "{{ etcd_peer_port1 }}"
     - "{{ etcd_peer_port2 }}"
 
-#  Save rules into /etc/sysconfig/iptables file for restoring rules on boot 
+#  Save rules into /etc/sysconfig/iptables file for restoring rules on boot
 - name: Save iptables
   command: service iptables save
```

Comment:
I think we should use systemctl instead ... , but this may still work on older version of rhel